markdown/ch1.markdown

```diff
@@ -74,23 +81,22 @@ Therefore, we usually measure mRNA levels and base our analyses on that.
 As you will see below, it often doesn't matter, because we are using mRNA levels for their power to predict biological outcomes, rather than to make specific statements about proteins.
 
 ![Central Dogma of Molecular Biology](../figures/central_dogma.png)
-**[ED NOTE doodle by Juan Nunez-Iglesias]**
 
 It's important to note that the DNA in every cell of your body is identical.
-Thus, the differences between cells arise from *differential expression* of that DNA into RNA.
-Similarly, as we shall see in this chapter, differential expression can distinguish different kinds of cancer.
+Thus, the differences between cells arise from *differential expression* of
+that DNA into RNA: in different cells, different parts of the DNA are processed
+into downstream molecules. Similarly, as we shall see in this chapter and the
+next, differential expression can distinguish different kinds of cancer.
 
 ![Gene expression](../figures/differential_gene_expression.png)
-**[ED NOTE doodle by Juan Nunez-Iglesias]**
 
 The state-of-the-art technology to measure mRNA is RNA sequencing (RNAseq).
-RNA is extracted from a tissue, for example from a biopsy from a patient, *reverse transcribed* back into DNA (which is more stable), and then read out using chemically modified bases that glow when they are incorporated into the DNA sequence.
+RNA is extracted from a tissue sample, for example from a biopsy from a patient, *reverse transcribed* back into DNA (which is more stable), and then read out using chemically modified bases that glow when they are incorporated into the DNA sequence.
 Currently, high-throughput sequencing machines can only read short fragments (approximately 100 bases is common). These short sequences are called “reads”.
 We measure millions of reads and then based on their sequence we count how many reads came from each gene.
-For this chapter we’ll be starting directly from this count data, but in [ch7?] we will talk more about how this type of data can be determined.
+For this chapter we’ll be starting directly from this count data.
```

Comment:
I've removed "for this chapter", but left the starting point comment since it tells people what to expect (ie don't worry about all the sequence or "millions of reads")